runtime/lua/vim/lsp/codelens.lua

```diff
@@ -279,25 +280,35 @@ function M.on_codelens(err, result, ctx, _)
   end)
 end
 
---- Refresh the codelens for the current buffer
+--- @class vim.lsp.codelens.RefreshOptions
+--- @field bufnr integer? filter by buffer. All buffers if nil, 0 for current buffer
+
+--- Refresh the lenses.
 ---
 --- It is recommended to trigger this using an autocmd or via keymap.
 ---
 --- Example:
 ---
 --- ```vim
---- autocmd BufEnter,CursorHold,InsertLeave <buffer> lua vim.lsp.codelens.refresh()
+--- autocmd BufEnter,CursorHold,InsertLeave <buffer> lua vim.lsp.codelens.refresh(0)
 --- ```
-function M.refresh()
+---
+--- @param opts? vim.lsp.codelens.RefreshOptions
```

Comment:
I decided to use a different type here instead of the existing `lsp.util.RefreshOptions` since I'm not exactly sure we want to support all options here as well.